functions/awsUploader/index.js

```diff
@@ -10,34 +10,14 @@ exports.createHandler = async (snap, bucket, aws, context) => {
   const metadata = snap.data();
   const durationSeconds = Math.ceil(metadata.duration);
   const fileSize = metadata.size;
-  const params = {
```

Comment:
What does it mean that we've lost duration and the expiration field? Do you know how those were being used before?